framework/source/class/qx/event/handler/DragDrop.js

```diff
@@ -613,7 +615,9 @@ qx.Class.define("qx.event.handler.DragDrop",
       }
 
       // find current hovered droppable
-      var el = e.getTarget();
+      // TODO: Review this (before the fix it was "e.getTarget()", but I feel this to be better)
+      var doc = this.__manager.getWindow().document;
+      var el = doc.elementFromPoint(e.getDocumentLeft(), e.getDocumentTop());
       var cursor = this.getCursor();
```

Comment:
does it only not work when the iframe content is coming from another domain ... that would explain why your solution works as it does not look at the content of the iframe